src/Session.ts

```diff
@@ -5,10 +5,13 @@ import type { Context, Middleware } from '../deps.ts'
 import type Store from './stores/Store.ts'
 import type { CookiesGetOptions, CookiesSetDeleteOptions } from '../deps.ts'
 
+export type SessionCookieName = string;
+
 interface SessionOptions {
   expireAfterSeconds?: number | null
   cookieGetOptions?: CookiesGetOptions
   cookieSetOptions?: CookiesSetDeleteOptions
+  sessionCookieName?: SessionCookieName
```

Comment:
Not sure we need a special type for this, `sessionCookieName?: string` should be fine.